docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,64 @@
+<button uxFocusIndicator
+        tabindex="0"
+        class="icon-demo btn btn-link button-secondary btn-text"
+        [uxPopover]="iconCodeSnippet"
+        popoverClass="icon-code-snippet-popover"
+        placement="bottom">
+    <i class="{{ iconClass }} p-r-xs" [ngClass]="classname"></i>
+    <span>{{ iconset === 'ux-icons' ? name : classname }}</span>
+</button>
+
+<ng-template #iconCodeSnippet>
+    <table class="icon-code-snippets-table">
+        <tbody>
+            <tr *ngIf="iconset === 'ux-icons'">
+                <td class="p-l-xs">
+                    <code>
+                        &lt;ux-icon name="{{name}}"&gt;&lt;/ux-icon&gt;
+                    </code>&nbsp;&nbsp;
+                </td>
+                <td>
+                    <i class="ux-icon ux-icon-clipboard m-r-xs"
+                       [uxTooltip]="tooltipTemplate"
+                       (click)="copy('ux-icon-component')">
```

Comment:
If something is a button, then it should be a `button`. We have styles for a [borderless icon-only button](https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/ui-components/buttons#icon-buttons-documentation-component) with proper mouseover and focus support, so let's leverage that here.

Reply:
changed